plugin/src/main/java/org/astropeci/omw/item/PeriodicItemDispenser.java

```diff
@@ -107,6 +107,14 @@ private void giveItemsPeriodically() {
 
             if (!inventory.contains(item.getType()) && inventory.getItemInOffHand().getType() != item.getType()) {
                 inventory.addItem(item);
+            } else if(item.getType() == Material.ARROW) {
```

Comment:
```suggestion
            } else if (item.getType() == Material.ARROW) {
```
Slight formatting typo, see the main comment for other issues.